engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -21,7 +20,7 @@ import scala.util.{Failure, Success, Try}
 trait SwaggerService extends SwaggerUiResourceHttpService {
   override def swaggerServiceName = "cromwell"
 
-  override def swaggerUiVersion = "2.1.1"
+  override def swaggerUiVersion = "2.2.2"
```

Comment:
Revert this back to `2.1.1`. Due to a bug fixed in swagger-ui 3.x, this causes rendering issues:

![screen shot 2017-03-28 at 4 25 15 pm](https://cloud.githubusercontent.com/assets/791985/24425785/6596e652-13d3-11e7-8e03-6d57bcae7ee8.png)

Unfortunately swagger-ui 3.x has its own issues with our setup that would need to be debugged.